benches/README.md

```diff
@@ -54,3 +54,76 @@ To run the benchmarks for Rescue Prime, Poseidon and SHA3, clone the following [
 ```
 cargo bench hash
 ```
+
+# Miden VM DSA
```

Comment:
I would probably have only one top level heading (e.g., something like `Benchmarks`) and then have `Hash functions` and `Digital signatures` as level 2 headings (i.e., with `##`). Heading level of all other sections would also be increased by 1.

Reply:
Done